skeletons/web-extension/manifest.json

```diff
@@ -39,8 +43,8 @@
 
   "action": {
     "default_icon": {
-      "19": "{{PANE_ROOT}}/assets/images/icon19.png",
-      "38": "{{PANE_ROOT}}/assets/images/icon38.png"
+      "19": "{{PANE_ROOT}}/assets/images/icon19_grey.png",
```

Comment:
Why are the default icons the grey ones?

Reply:
by default it should show `not detected` icon. as the detection state only shows when an ember app is detected